paasta_tools/clusterman.py

```diff
@@ -0,0 +1,17 @@
+import staticconf
+
+CLUSTERMAN_YAML_FILE_PATH = '/nail/srv/configs/clusterman.yaml'
+CLUSTERMAN_METRICS_YAML_FILE_PATH = '/nail/srv/configs/clusterman_metrics.yaml'
+
+
+def get_clusterman_metrics():
+    try:
+        import clusterman_metrics
+        clusterman_yaml = CLUSTERMAN_YAML_FILE_PATH
+        staticconf.YamlConfiguration(CLUSTERMAN_METRICS_FILE_PATH, namespace='clusterman_metrics')
+    except ImportError:
+        # our cluster autoscaler is not currently open source, sorry!
+        clusterman_metrics = None
+        clusterman_yaml = None
+
+    return clusterman_metrics, clusterman_yaml
```

Comment:
Can we get away without returning the YAML at all? That would simplify this function.

Reply:
@stug needs the yaml for the spark_run stuff, idk if there's a better way to do it.

Reply:
I need it in spark-run, but we could possibly put logic in spark-run so that it checks for the existence of clusterman_metrics and then uses the YAML file path (imported from this module) if it can use it (i.e. if clusterman_metrics was imported)?